src/extensions/snippets/SnippetsView/SnippetBox.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent, useCallback, useState } from 'react';
+import React, { FunctionComponent, useCallback, useMemo, useState } from 'react';
```

Comment:
The changes in this file somehow did not get merged into master - they synchronize the current timepoint with the snippets view